lib/widgets/compose_box.dart

```diff
@@ -1343,6 +1344,20 @@ class _SendButtonState extends State<_SendButton> {
         message: message);
       return;
     }
+
+    store = PerAccountStoreWidget.of(context);
+    final destination = widget.getDestination();
+    if (
+      destination is StreamDestination
+      && store.subscriptions[destination.streamId] == null
+    ) {
+      // We don't get new-message events for unsubscribed channels,
+      // but we can refresh the view when a send-message request succeeds,
```

Comment:
I think this could use a line setting the context a bit more broadly:
```suggestion
    ) {
      // The message was sent to an unsubscribed channel.
      // We don't get new-message events for unsubscribed channels,
      // but we can refresh the view when a send-message request succeeds,
```